mistletoe/html_renderer.py

```diff
@@ -205,11 +205,11 @@ def render_document(self, token: block_token.Document) -> str:
 
     @staticmethod
     def escape_html(raw: str) -> str:
-        return html.escape(html.unescape(raw)).replace('&#x27;', "'")
+        return html.escape(raw).replace('&#x27;', "'")
 
     @staticmethod
     def escape_url(raw: str) -> str:
         """
         Escape urls to prevent code injection craziness. (Hopefully.)
         """
-        return html.escape(quote(html.unescape(raw), safe='/#:()*?=%@+,&;'))
+        return html.escape(quote(raw, safe='/#:()*?=%@+,&;'))
```

Comment:
OK, so I believe we should remove the surplus call to `unescape()` in the other `escape_url()` definitions too, shouldn't we? (The best would probably be to get rid of the duplicate definitions, but maybe leave that for #114...?)